pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/reporter/BrokerLoadDataReporter.java

```diff
@@ -152,4 +180,49 @@ protected double percentChange(final double oldValue, final double newValue) {
         }
         return 100 * Math.abs((oldValue - newValue) / oldValue);
     }
+
+    @VisibleForTesting
+    protected void tombstone() {
+        var now = System.currentTimeMillis();
+        if (now - lastTombstonedAt < tombstoneDelayInMillis) {
+            return;
+        }
+        var lastSuccessfulTombstonedAt = lastTombstonedAt;
+        lastTombstonedAt = now; // dedup first
+        brokerLoadDataStore.removeAsync(lookupServiceAddress)
+                .whenComplete((__, e) -> {
+                            if (e != null) {
+                                log.error("Failed to clean broker load data.");
+                                lastTombstonedAt = lastSuccessfulTombstonedAt;
+                            } else {
+                                boolean debug = ExtensibleLoadManagerImpl.debug(conf, log);
+                                if (debug) {
+                                    log.info("Cleaned broker load data.");
+                                }
+                            }
+                        }
+                );
+
+    }
+
+    @Override
+    public void handleEvent(String serviceUnit, ServiceUnitStateData data, Throwable t) {
+        executor.execute(() -> {
```

Comment:
Is there any reason to use executor here? I did not see any block operating here.